testing/scripts/kind_test_all.sh

```diff
@@ -34,13 +34,37 @@ export KUBECONFIG=$(kind get kubeconfig-path)
 
 # ONLY RUN THE FOLLOWING IF SUCCESS
 if [[ ${KIND_EXIT_VALUE} -eq 0 ]]; then
-    # BUILD S2I BASE IMAGES
-    make s2i_build_base_images
-    S2I_EXIT_VALUE=$?
 
-    # CREATE PROTOS
-    make build_protos
-    PROTOS_EXIT_VALUE=$?
+    echo "Files changed in python folder:"
+    git --no-pager diff --exit-code --name-only origin/master ../../python
+    PYTHON_MODIFIED=$?
```

Comment:
Would it make sense to also test, besides changes, if the latest image version is present?

Reply:
Good point - in this case it would be using vXXX-SNAPSHOT tho, so it would be whatever was the latest in master. This does raise the questions on whether we'd want to use latest master or latest tag...